requirements.txt

```diff
@@ -3,15 +3,15 @@ cycler==0.10.0
 kiwisolver==1.1.0
 matplotlib==3.1.3
 numpy==1.18.1
-opencv-python==3.3.1
+opencv-python==3.4.2.17
+openpyxl>=2.6.1
 pandas==1.0.2
 pyparsing==2.4.6
 python-dateutil==2.8.1
 pytz==2019.3
+scikit-image>=0.16.2
+scipy==1.1.0
 six==1.14.0
 tabulate==0.8.3
 tornado==6.0.4
```

Comment:
pyparsing, python-dateutil, pytz, six, and tornado are all never used (my Pycharm can't find any imports or usage anywhere)

tabulate is used in a single file (visualize_elisa_spots.py) and a single print statement (line 45).  Perhaps we can remove this?